game/video_driver.hh

```diff
@@ -8,6 +8,7 @@
 
 unsigned int screenW();
 unsigned int screenH();
+const unsigned int targetWidth();
```

Comment:
i'm getting this warning:
`warning: type qualifiers ignored on function return type [-Wignored-qualifiers] const unsigned int targetWidth();`